app/src/components/Interfaces.tsx

```diff
@@ -0,0 +1,17 @@
+// this file actually might not be necessary.
+// bc in CardGrid.tsx I had to use AllAnimalsQuery type instead of any of this
```

Comment:
Or we merge it together with global types?
No matter what I think that files like this should be in the root of /src and not as a part of components.
And as a rule of thumb I think that interfaces/types that only are used in one place does not need to be stored globally but can reside along the code where it is being used.